client/lib/analytics/utils/may-we-session-track.ts

```diff
@@ -0,0 +1,14 @@
+import { isRegionInStsZone } from '@automattic/calypso-analytics';
+import cookie from 'cookie';
+
+const mayWeSessionTrack = (): boolean => {
+	const cookies = cookie.parse( document.cookie );
+
+	if ( isRegionInStsZone( cookies.country_code, cookies.region ) ) {
+		return false;
+	}
+
+	return ! isRegionInStsZone( cookies.country_code, cookies.region );
```

Comment:
I think we can remove this `if ( isRegionInStsZone(...) ) {` block, right?  The return statement below it will return the correct result without the `if` block being there, I believe.. right?

Reply:
You are correct, it looks like I decided to simplify the function and didn't complete the job, thank you for the catch!